packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/examples/PeoplePicker.Types.Example.tsx

```diff
@@ -144,6 +144,7 @@ export class PeoplePickerTypesExample extends BaseComponent<any, IPeoplePickerEx
           onFocus: (ev: React.FocusEvent<HTMLInputElement>) => console.log('onFocus called')
         } }
         componentRef={ (component: IBasePicker<IPersonaProps>) => this._picker = component }
+        wellAriaLabel={ 'People Picker' }
```

Comment:
Since this is just the example, in non-English cases they could pass in a different value for this prop.